tools/notification-configuration/github-codeowner-subscriber/Program.cs

```diff
@@ -52,22 +46,18 @@ class Program
             }))
 #pragma warning restore CS0618 // Type or member is obsolete
             {
-                var devOpsService = AzureDevOpsService.CreateAzureDevOpsService(
-                    Environment.GetEnvironmentVariable(devOpsTokenVar),
+                var devOpsService = AzureDevOpsService.CreateAzureDevOpsService(devOpsTokenVar,
                     $"https://dev.azure.com/{organization}/",
                     loggerFactory.CreateLogger<AzureDevOpsService>()
                 );
                 
                 var gitHubServiceLogger = loggerFactory.CreateLogger<GitHubService>();
                 var gitHubService = new GitHubService(gitHubServiceLogger);
 
-                var githubNameResolver = new GitHubNameResolver(
-                    Environment.GetEnvironmentVariable(aadAppIdVar),
-                    Environment.GetEnvironmentVariable(aadAppSecretVar),
-                    Environment.GetEnvironmentVariable(aadTenantVar),
-                    Environment.GetEnvironmentVariable(kustoUrlVar),
-                    Environment.GetEnvironmentVariable(kustoDatabaseVar),
-                    Environment.GetEnvironmentVariable(kustoTableVar),
+                var githubToAadResolver = new GitHubToAadConverter(
+                    aadAppIdVar,
```

Comment:
We should probably create the credential option outside of this class and pass it in.